dev/Dockerfile

```diff
@@ -16,8 +16,11 @@ RUN apt-get update && \
         php5.6-cli \
         php5.6-imagick \
         php5.6-mbstring \
+        php5.6-soap \
         supervisor
 
+ENV TERM xterm
```

Comment:
No need for this. This container will not typically be started with a TTY.

Reply:
Well maybe not typically, but if you need to `docker exec -it` into the container because you want to change something, you might wonder why the terminal does not work. Having it set container wide means you don't need to set it again every time you open it.